test/setup/setup.js

```diff
@@ -5,7 +5,7 @@ module.exports = function() {
   Backbone.$ = jQuery;
   Backbone.ChildViewContainer = require('backbone.babysitter');
   Backbone.Radio = require('backbone.radio');
-  var Marionette = require('../../src/backbone.marionette');
+  var Marionette = require('../../src/backbone.marionette').default;
```

Comment:
I agree it is annoying.. not sure about needing the plugin at the moment.  If it comes up again for some reason, certainly.